spec/request/service_instances_spec.rb

```diff
@@ -1260,7 +1260,10 @@ def check_filtered_instances(*instances)
             expect(last_response).to have_status_code(422)
             expect(parsed_response['errors']).to include(
               include({
-                        'detail' => 'Invalid service plan. Ensure that the service plan exists, is available, and you have access to it.',
+                        'detail' => 'Invalid service plan. This could be due to a space-scoped broker which is offering the service plan ' \
```

Comment:
Would be good to have the same tests for the update case (PATCH). If we had those, above bug which results in a 500 would have been detected.

Reply:
Good point, will add it